packages/components/buttons/icon-button/src/icon-button.styles.ts

```diff
@@ -57,8 +57,8 @@ const getSizeStyles = (size: TIconButtonProps['size']) => {
   switch (size) {
     case 'small':
       return css`
-        height: ${designTokens.heightForButtonAsSmall};
-        width: ${designTokens.heightForButtonAsSmall};
+        height: 16px;
```

Comment:
**nit**:
In this case, since we have tokens for the other button sizes, I would suggest to keep a token for the small size as well.
Even if we are just using it here, perhaps we will need to adjust other buttons and it will also helps us to keep consistency here and maintainability.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2785/commits/065e27db2ab90e9c6103bb78679b8c7d9ee5283f
Thank you